src/dags/beheerkaart.py

```diff
@@ -104,11 +128,13 @@
         distribution_id="1",
     )
 
-     # 10. Grant database permissions
-    grant_db_permissions = PostgresPermissionsOperator(
-        task_id="grants",
-        dag_name=dataset_name
-    )
+    # 10. Grant database permissions
+    grant_db_permissions = PostgresPermissionsOperator(task_id="grants", dag_name=dataset_name)
 
 # FLOW
-slack_at_start >> drop_tables >> swift_load_task >> provenance_renames >> swap_schema >> mkdir >> create_geopackage >> zip_geopackage >> upload_data >> grant_db_permissions  # noqa
+slack_at_start >> drop_tables >> create_tables
+
+for table in create_tables:
+    table >> swift_load_task
+
+swift_load_task >> provenance_renames >> swap_schema >> mkdir >> create_geopackage >> zip_geopackage >> upload_data >> grant_db_permissions  # noqa
```

Comment:
Wellicht hier nog even haakjes omheen, dat leest wat prettiger, omdat het vertikaal geformatteerd wordt en is het beter ivm. diffen.